src/twisted/words/protocols/jabber/xmlstream.py

```diff
@@ -400,21 +401,39 @@ class TLSInitiatingInitializer(BaseFeatureInitiatingInitializer):
     set the C{wanted} attribute to False instead of removing it from the list
     of initializers, so a proper exception L{TLSRequired} can be raised.
 
-    @cvar wanted: indicates if TLS negotiation is wanted.
+    @ivar wanted: indicates if TLS negotiation is wanted.
     @type wanted: C{bool}
+
+    @ivar contextFactory: An object which creates appropriately configured TLS
```

Comment:
Same question here about attribute name…

Reply:
Also, does this really need to be a public attribute, or should it just be an `__init__` parameter?

Reply:
I will change this to be private indeed.